core/domain/summary_services.py

```diff
@@ -396,3 +396,20 @@ def get_top_rated_exploration_summary_dicts(language_codes):
 
     return _get_displayable_exp_summary_dicts(
         sorted_exp_summaries, include_contributors=False)
+
+
+def get_recently_published_exploration_summary_dicts():
+    """Returns a list of recently published explorations
+     with the given language code.
+    """
+    recently_published_exploration_summaries = [
+        exp_summary for exp_summary in
+        exp_services.get_recently_published_exploration_summaries().values()]
+
+    summary_dicts = sorted(recently_published_exploration_summaries,
+                           key=lambda exp_summary:
+                           exp_summary.exp_model_first_published_msec,
+                           reverse=True)
```

Comment:
Just wondering, do you still need to sort here (if the query returns results that are already ordered)?